iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -201,6 +200,15 @@ public List<InputSplit> getSplits(JobContext context) {
         .ofNullable(HiveIcebergStorageHandler.table(conf, conf.get(InputFormatConfig.TABLE_IDENTIFIER)))
         .orElseGet(() -> Catalogs.loadTable(conf));
 
+    if (conf.get(InputFormatConfig.TABLE_IDENTIFIER) == null) {
+      conf.set(InputFormatConfig.TABLE_IDENTIFIER, table.name());
```

Comment:
I think we do not need this param. We can just use` table.name()`,

Reply:
also we can skip the if check as well

Reply:
yes, this is also what i mean ;)